include/boost/winapi/process.hpp

```diff
@@ -68,7 +71,20 @@ BOOST_SYMBOL_IMPORT boost::winapi::BOOL_ BOOST_WINAPI_WINAPI_CC CreateProcessA(
     ::_PROCESS_INFORMATION* lpProcessInformation);
 #endif
 
-BOOST_SYMBOL_IMPORT boost::winapi::BOOL_ BOOST_WINAPI_WINAPI_CC CreateProcessW(
+#if defined(_WIN32_WCE)
+BOOST_WINAPI_IMPORT_EXCEPT_WM boost::winapi::BOOL_ BOOST_WINAPI_WINAPI_CC CreateProcessW(
+	boost::winapi::LPCWSTR_ lpApplicationName,
+	boost::winapi::LPCWSTR_ lpCommandLine,
+	::_SECURITY_ATTRIBUTES* lpProcessAttributes,
+	::_SECURITY_ATTRIBUTES* lpThreadAttributes,
+	boost::winapi::INT_ bInheritHandles,
+	boost::winapi::DWORD_ dwCreationFlags,
+	boost::winapi::LPVOID_ lpEnvironment,
+	boost::winapi::LPWSTR_ lpCurrentDirectory,
```

Comment:
Here and in other functions, changing `LPCWSTR_` to `LPWSTR_` is not portable as it breaks user's code which passes constant strings as arguments. If this function actually does not modify the argument, there needs to be an inline wrapper function in `boost::winapi` which will cast away the constness.

Reply:
agree